src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java

```diff
@@ -284,10 +284,9 @@ private void testAjaxPictureUrl() {
     private <T extends AppPage> T getProfilePicturePage(String instructorId, String email, String courseId,
                                                         Class<T> typeOfPage) {
         AppUrl profileUrl = createUrl(Const.ActionURIs.STUDENT_PROFILE_PICTURE)
-                                   .withUserId(testData.accounts.get(instructorId).googleId)
                                    .withParam(Const.ParamsNames.STUDENT_EMAIL, email)
                                    .withParam(Const.ParamsNames.COURSE_ID, courseId);
-        return loginAdminToPage(profileUrl, typeOfPage);
+        return loginInstructorToPage(instructorId, profileUrl, typeOfPage);
```

Comment:
@tran-tien-dat this is the root cause of #8393. With this, in staging server you need a legitimate Google ID (a few, in fact) in order for this method call to succeed. If logged in as admin like previously done, masquerade mode is activated and thus a single admin Google ID suffices.

cc @whipermr5

Reply:
I think I can fix this by using the instructor account defined in `TestProperties`:

https://github.com/TEAMMATES/teammates/blob/4f57a6e62cc1e73000daecb05274b714af5c990b/src/test/java/teammates/test/driver/TestProperties.java#L32-L36

Will this solve the problem?

Reply:
In principle it will, but will also create new problems since there are at least 2 different instructors logging in in this test case, and adding a new test instructor has quite a widespread effect.
To make life easier, the best course of action will be to revert back to masquerade first, and then going down that path if you really need to.

Reply:
Hmm. Changing from masquerade mode to actual instructor login is kind of necessary in this PR's approach. I think I can change the logic in the test so that it tests for the same thing but only require 1 instructor account to be logged in. (i.e. create another course for instructor 2, and then instead of instructor 2 trying to view pics from instructor 1's course, test instructor 1 trying to view pics from instructor 2's course).

Reply:
Ok, that sounds like it's going to work.

Reply:
Apologies for not catching this 😅

Reply:
@whipermr5 @tran-tien-dat don't worry, I'd do this kind of mistake myself.